python/dgl/heterograph.py

```diff
@@ -2254,49 +3231,75 @@ def out_edges(self, u, form='uv', etype=None):
         else:
             raise DGLError('Invalid form: {}. Must be "all", "uv" or "eid".'.format(form))
 
-    def all_edges(self, form='uv', order=None, etype=None):
-        """Return all edges with the specified type.
+    def all_edges(self, form='uv', order='eid', etype=None):
+        """Return all edges with the specified edge type.
 
         Parameters
         ----------
         form : str, optional
-            The return form. Currently support:
-
-            - ``'eid'`` : one eid tensor
-            - ``'all'`` : a tuple ``(u, v, eid)``
-            - ``'uv'``  : a pair ``(u, v)``, default
-        order : str or None
-            The order of the returned edges. Currently support:
-
-            - ``'srcdst'`` : sorted by their src and dst ids.
-            - ``'eid'``    : sorted by edge Ids.
-            - ``None``     : arbitrary order, default
+            The return form, which can be one of the following:
+
+            - ``'eid'``: The returned result is a 1D tensor :math:`EID`, representing
+              the IDs of all edges.
+            - ``'uv'`` (default): The returned result is a 2-tuple of 1D tensors :math:`(U, V)`,
+              representing the source and destination nodes of all edges. For each :math:`i`,
+              :math:`(U[i], V[i])` forms an edge.
+            - ``'all'``: The returned result is a 3-tuple of 1D tensors :math:`(U, V, EID)`,
+              representing the source nodes, destination nodes and IDs of all edges.
+              For each :math:`i`, :math:`(U[i], V[i])` forms an edge with ID :math:`EID[i]`.
+        order : str, optional
+            The order of the returned edges, which can be one of the following:
+
+            - ``'srcdst'``: The edges are sorted first by their source node IDs and then
+              by their destination node IDs to break ties.
```

Comment:
if multigraph, the edges will be sorted by eids or arbitrary?

Reply:
I think it will be arbitrary and it should be so.